src/main/java/org/nixos/idea/psi/NixStringLiteralEscaper.kt

```diff
@@ -0,0 +1,33 @@
+package org.nixos.idea.psi
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.LiteralTextEscaper
+import com.intellij.psi.PsiLanguageInjectionHost
+import org.nixos.idea.psi.impl.AbstractNixString
+import org.nixos.idea.util.NixIndStringUtil
+import org.nixos.idea.util.NixStringUtil
+
+class NixStringLiteralEscaper(host: AbstractNixString) : LiteralTextEscaper<PsiLanguageInjectionHost>(host) {
+
+    override fun isOneLine(): Boolean = false
```

Comment:
I took the example from [HCL](https://github.com/JetBrains/intellij-plugins/blob/fb9094045d7492228087a091f2ec248f77825dd8/terraform/src/org/intellij/terraform/hcl/psi/impl/HCLStringLiteralTextEscaper.kt#L12) which also always returns false. I will try to implement proper smart logic, see if that makes any difference, but I suspected it might be best to copy my reference for now 🤷